src/Entity/ReactionRule.php

```diff
@@ -135,9 +135,6 @@ public function getExpression() {
   /**
    * Returns the Rules expression manager.
    *
-   * @todo Actually we should use dependency injection here, but is that even
-   *   possible with config entities? How?
```

Comment:
Hm, should we remove that todo? We are using \Drupal here, which we should not do, so we should document why we do it.